app/code/core/Mage/Sales/Block/Billing/Agreements.php

```diff
@@ -86,20 +86,20 @@ public function getItemValue(Mage_Sales_Model_Billing_Agreement $item, $key)
             case 'created_at':
             case 'updated_at':
                 $value = ($item->getData($key))
-                    ? $this->helper('core')->formatDate($item->getData($key), 'short', true) : $this->__('N/A');
+                    ? $this->formatDate($item->getData($key), 'short', true) : $this->__('N/A');
                 break;
             case 'edit_url':
                 $value = $this->getUrl('*/billing_agreement/view', array('agreement' => $item->getAgreementId()));
                 break;
             case 'payment_method_label':
                 $label = $item->getAgreementLabel();
-                $value = ($label) ? $label : $this->__('N/A');
+                $value = ($label) ?: $this->__('N/A');
                 break;
             case 'status':
                 $value = $item->getStatusLabel();
                 break;
             default:
-                $value = ($item->getData($key)) ? $item->getData($key) : $this->__('N/A');
+                $value = ($item->getData($key)) ?: $this->__('N/A');
```

Comment:
Useless parenthesis:
```suggestion
                $value = $item->getData($key) ?: $this->__('N/A');
```